javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -291,15 +278,6 @@ private static void installEarlyInstrumentation(
     agentBuilder.installOn(instrumentation);
   }
 
-  private static void copyNecessaryConfigToSystemProperties(ConfigProperties config) {
-    for (String property : asList("otel.instrumentation.experimental.span-suppression-strategy")) {
-      String value = config.getString(property);
-      if (value != null) {
-        System.setProperty(property, value);
-      }
-    }
-  }
```

Comment:
this isn't needed anymore now that span_suppression_strategy is read first from declarative config API